src/poprox_recommender/default.py

```diff
@@ -1,28 +1,36 @@
 # pyright: basic
 import logging
+from dataclasses import dataclass
 from typing import Any
 
-from poprox_concepts import ArticleSet, InterestProfile
+from poprox_concepts import Article, ArticleSet, InterestProfile
 from poprox_recommender.components.diversifiers import MMRDiversifier, PFARDiversifier, TopicCalibrator
 from poprox_recommender.components.embedders import ArticleEmbedder, UserEmbedder
 from poprox_recommender.components.filters import TopicFilter
 from poprox_recommender.components.rankers.topk import TopkRanker
 from poprox_recommender.components.samplers import UniformSampler
 from poprox_recommender.components.scorers import ArticleScorer
+from poprox_recommender.lkpipeline import Pipeline
 from poprox_recommender.model import get_model
-from poprox_recommender.pipeline import PipelineState, RecommendationPipeline
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.DEBUG)
 
 
+@dataclass
+class Recommendations:
+    recs: list[Article]
+    initial: ArticleSet | None = None
+    reranked: ArticleSet | None = None
```

Comment:
This gives me pause because I don't think reranking is the only pipeline step that we may be interested in before and after states of. One of the advantages of `PipelineState` was that you could design your pipeline any way you wanted and still grab particular component outputs by name